.github/scripts/snapshot_update_pr.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# This script is meant to be run as a part of a workflow 
+# which sets the necessary environment variables
+
+pushd "$REPO"
+# need a tmp file for updating Schutzfile
```

Comment:
I just wonder if lines 7-33 aren't worth a small python script. I completely get that `jq` is powerful but the readability is just hard. I can imagine that this would be slightly more comprehensive when written in a more imperative style (e.g. line 28).

What are your thougths?

Reply:
My thoughts are that I have no idea how this would look in Python but you're probably right and I'll try it to see how it will look like :smile:

Reply:
Well I again created "something"...  Tbh I don't think it's much simpler, but maybe I just suck at python :D looking forward to another review. I kept both commits just for comparison sake, I'll squash them before merge :)